src/Plugin/InventoryInStorePickupSales/Model/Order/IsFulfillablePlugin.php

```diff
@@ -0,0 +1,14 @@
+<?php
+declare(strict_types=1);
+
+namespace Ampersand\DisableStockReservation\Plugin\InventoryInStorePickupSales\Model\Order;
+
+use Magento\InventoryInStorePickupSales\Model\Order\IsFulfillable;
+
+class IsFulfillablePlugin
+{
+    public function afterExecute(IsFulfillable $subject, bool $result): bool
```

Comment:
Hey @sprankhub and @waleedbasit 

So I've not used the instore pickup at all so apologies if this is a series of silly question.

I can see the main execute function is defined like so

https://github.com/magento/inventory/blob/c4e3a4ef40ed720e2dd6b50156eb1ae1abb98162/InventoryInStorePickupSales/Model/Order/IsFulfillable.php#L68-L84

Ideally all we'd need to change is the `isItemFulfillable` but we cannot plugin on the private method, that's why we're doing it this way?

And because we have disabled the stock reservation process, we not essentially need to shortcircuit `isItemFulfillable` to always say `true` because the quantity has been deducted already at the point of the order creation, that's what this accomplishes?

Reply:
> Ideally all we'd need to change is the `isItemFulfillable` but we cannot plugin on the private method, that's why we're doing it this way?

Exactly.

> we not essentially need to shortcircuit `isItemFulfillable` to always say `true`

Not sure if I follow here completely. I think the word "not" is wrong here. We do need to short-circuit `isItemFulfillable` to always say `true`, because the qty has already been deducted at the point of the order creation.

Reply:
`not` should have read as `now` 😬 #dyslexia 

this makes sense, unfortunately I dont have any UI driven tests of the admin panel but i'll see what i can spin up, maybe an integration test will be suitable for this scenario